conda/recipes/ast_canopy/meta.yaml

```diff
@@ -20,29 +20,23 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}
+    - {{ stdlib('c') }}
     - cmake >=3.28
-    - pybind11
-    - clangdev >=18
     - ninja
-    - sysroot_{{ target_platform }} 2.17
   host:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - cuda-version {{ cuda_compiler_version }}
     - python
+    - pybind11
     - pip
     - scikit-build-core
+    - clangdev >=18
     - llvmdev >=14
     - llvm >=14
   run:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - cuda-version >=12.5
+    - cuda-nvcc >=12.5
+    - cuda-nvcc_{{ target_platform }} >=12.5
```

Comment:
```suggestion
```

As mentioned in comment, the cuda-nvcc spec with the version lower bound should be enough

Reply:
I believe we added this because the solver would not provide valid environments otherwise. Hands-on tests will be needed to find a root cause.

Reply:
Revisiting this - I thought with `cuda-nvcc=12.5+` requirement the support should be complete?